esx_service/utils/auth_api.py

```diff
@@ -335,12 +335,11 @@ def _tenant_create(name, description="", vm_list=None, privileges=None):
             return error_info, None
 
         logging.debug("_tenant_create: vms=%s", vms)
-        vms_uuid_list = [(vm_id) for (vm_id, vm_name) in vms]
 
     error_info, tenant = create_tenant_in_db(
         name=name,
         description=description,
-        vms=vms_uuid_list,
+        vms=vms,
```

Comment:
Is it ok to init vms as [] vs. None? Code thats called from here to create the tenant use vms as a list and don't ever check the list is None. While its done in a try-except block in auth_data.py:create_tenant(), the logs may seem ambigous why an exception happens on tenant create.